indico/migrations/versions/20171117_1035_40d4569835b8_add_include_description_to_reminders.py

```diff
@@ -0,0 +1,27 @@
+"""Add include_description to reminders
+
+Revision ID: 40d4569835b8
+Revises: 
+Create Date: 2017-11-17 10:35:12.378432
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '40d4569835b8'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('reminders', sa.Column('include_description', sa.Boolean(), nullable=False, default=False,
+	 server_default='false'), schema='events')
+    op.alter_column('reminders', sa.Column('include_description', sa.Boolean(), nullable=False, default=False),
```

Comment:
the one thing that is actually needed here is missing: `server_default=None`
nothing changed on `nullable` so it doesn't need to be specified, and `default` is never needed in an alembic rev.

see https://github.com/indico/indico/blob/v1.9.11.dev18/indico/migrations/versions/201708301158_09fd62e56097_add_oauth_system_app_type.py#L42-L45 for an example on how to add a new column with a server-side default value during creation (for existing rows)

Reply:
Thanks, this actually helps how to do it right. Development.md was a bit cryptic in this respect.